client/scss/components/forms/_title.scss

```diff
@@ -1,19 +1,15 @@
 // Styles for title fields throughout the page.
 // The `title` class is meant to be applied by end users throughout different forms APIs.
-.w-panel.title,
-.w-field.title {
-  input,
-  textarea,
-  .Draftail-Editor .public-DraftEditor-content {
+:where(.w-panel, .w-field).title {
+  :where(input, textarea, .Draftail-Editor .public-DraftEditor-content) {
```

Comment:
Here and below – refactoring the selectors so they’re easier to work with, should have no incidence on the styles.